htdocs/product/card.php

```diff
@@ -666,7 +667,20 @@
 				if (count($object->errors)) {
 					setEventMessages($object->error, $object->errors, 'errors');
 				} else {
-					setEventMessages($langs->trans($object->error), null, 'errors');
+					if ($object->error == 'ErrorProductAlreadyExists') {
+						// permet une action personnallisée quand il y a tentative d'ajouter un produit
```

Comment:
Can you put the code comments in english please ?

Reply:
Oui, je vous envoie ça dans quelques instants, merci !